WookieeMatt/Essence20
# Rest notifications; `synth-en` to `synthEn`

##### In this PR
- Notifications now show up when resting
- Changing `synth-en` to `synthEn` to maintain casing
- Localizing Energon names in the template

##### Testing
- Click rest on different PCs
- For non-`canTransform` PCs: Should just display HP/stun resetting and rest completing
- For `canTransform` PCs:
  - Try different Energon values
  - Non-standard Energons should only appear in the noti string if they had a non-zero value before updating
